### PR TITLE
Move to openshift namespace

### PR DESCRIPTION
**Pull request by josefkarasek**

### CODE REVIEW

File: cmd/elasticsearch-operator/main.go [hunk 1]
```diff
@@ -30,7 +30,7 @@ func main() {
 	if err != nil {
 		logrus.Fatalf("Failed to get watch namespace: %v", err)
 	}
-	resyncPeriod := time.Duration(5)*time.Second
+	resyncPeriod := time.Duration(5) * time.Second
```
Comment (richm) [inline cmd/elasticsearch-operator/main.go]: Is this a gofmt thing?

**Comment by ewolinetz** [comment]
lgtm

**Comment by ewolinetz** [comment]
@jcantrill is ci properly building the images for this?

**Comment by ewolinetz** [comment]
/lgtm
tested it on my remote cluster, started up just fine no errors, created api objects as expected

**Pull request merged**